packages/core/components/progressBar/index.mdx

```diff
@@ -10,12 +10,42 @@ import ProgressBar from "./";
 
 # ProgressBar
 
+## Import
+`import { ProgressBar } from '@blasterjs/core'`
+
+## Basic Usage
+
 Displays a fractional amount of progress towards some end.
 
 <Playground>
   <ProgressBar value={25} max={100} />
 </Playground>
 
+## Striped and animated
+Use the `striped` and `animated` prop to change the default appearance and show motion.
```

Comment:
May be worth mentioning that `animated` has no visible effect unless `striped` is present.

Reply:
I added a callout